core/css.js

```diff
@@ -624,25 +624,40 @@ Blockly.Css.CONTENT = [
   '}',
 
   '.blocklyAngleCircle {',
-    'stroke: #444;',
+    'stroke: ' + Blockly.Colours.motion.tertiary + ';',
```

Comment:
Will the angle picker ever be used for anything other than motion blocks?  If so, it should take its colors from the parent block instead of having them set here.